docs/en/docs/configurations/staticfiles.md

```diff
@@ -41,3 +41,12 @@ but also via settings.
 ```
 
 This will make sure you keep the settings clean, separated and without a bloated **Esmerald** instance.
+
+
+## Multiple directories
+
+You have e.g. a node_modules directory you have to include? You can do so by passing multiple configurations
```

Comment:
Imagine, for example, you have multiple directories you would like to access including a `node_modules/` one. This is possible do do it by passing multiple configurations and shown below.